src/box/lua/config/instance_config.lua

```diff
@@ -493,6 +493,20 @@ return schema.new('instance_config', schema.record({
                     }),
                 }),
             }),
+            watchers = schema.record({
+                reconnect_timeout = schema.scalar({
+                    type = 'number',
+                    -- default = 1 is applied right in the
+                    -- etcd source. See a comment above
+                    -- regarding defaults in config.etcd.
+                }),
+                reconnect_max_attempts = schema.scalar({
+                    type = 'number',
```

Comment:
Nit: There is no reason to leave a number with a fractional part for an attempts count. So, I would change the type to `integer`.

Reply:
True, fixed.